code/modules/mechs/premade/heavy.dm

```diff
@@ -59,8 +59,8 @@
 	exosuit_desc_string = "heavy legs"
 	desc = "Exosuit actuators struggle to move these armored legs, and they're even worse at turning."
 	icon_state = "heavy_legs"
-	move_delay = 5
-	turn_delay = 2 // Turning should be easy , moving not.
+	move_delay = 7
```

Comment:
```suggestion
	move_delay = 5
```
If 5 is already really slow you could let it at that value (yet still nerf turning a bit)